src/components/dialogs/network-modifications/by-filter-deletion/by-filter-deletion.type.ts

```diff
@@ -22,7 +22,7 @@ export interface ByFilterDeletionEditData {
 }
 
 export interface ByFilterDeletionFormData {
-    type: keyof typeof EQUIPMENT_TYPES | null;
+    type: string | null;
```

Comment:
why did you set a more generic type for keys?

Reply:
Yeah wasn't too sure of this either there was an incompatibility with the associated `ByFilterDeletionDialog` yup schema which specified `yup.string()` for the field type, reverted and adapted yup schema to `yup.mixed<keyof typeof EQUIPMENT_TYPES>()` instead